### PR TITLE
Add PCI-DSS file rules

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:

- file_permissions_var_log_audit
- file_ownership_var_log_audit

### CODE REVIEW

**Review by cipherboy — APPROVED** [review]

File: linux_os/guide/system/auditing/auditd_configure_rules/file_ownership_var_log_audit/rule.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ identifiers:
     cce@rhel7: CCE-80125-8
     cce@rhel8: CCE-80808-9
     cce@rhel9: CCE-83726-0
+    cce@sle15: CCE-85802-7
```
Comment (cipherboy) [inline linux_os/guide/system/auditing/auditd_configure_rules/file_ownership_var_log_audit/rule.yml:21]: Note that this CCE is reused by an earlier PR:

```
linux_os/guide/system/permissions/files/permissions_important_account_files/file_owner_etc_group/rule.yml:17:    cce@sle15: CCE-85802-7
```

File: linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml [hunk 2]
```diff
@@ -21,6 +21,7 @@ identifiers:
     cce@rhel7: CCE-27205-4
     cce@rhel8: CCE-80819-6
     cce@rhel9: CCE-83720-3
+    cce@sle15: CCE-85801-9
```
Comment (cipherboy) [inline linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml:24]: Note that this CCE is reused by an earlier PR:

```
linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_etc_group/rule.yml:17:    cce@sle15: CCE-85801-9
```

**Review by cipherboy — CHANGES_REQUESTED** [review]
Looks like both CCEs added here are duplicates :/ Sorry for the spurious ACK.

File: linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml (hunk 2 above)
Comment (anivan-suse) [reply linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml:24]: Yup! My bad, fixing it now :) thanks

**Review by cipherboy — APPROVED** [review]
\o/ ty for the update

**Review by Mab879 — APPROVED** [review]

**Pull request merged**